README.md

```diff
@@ -100,6 +100,45 @@ streetwalrus@Akatsuki:~/source/shotgun(master)
 Further profiling has shown that the bottleneck in shotgun lies fully within the
 PNG encoder.
 
+### Going faster
```

Comment:
Mentions of the PAM format in the feature list/maim comparison should link to this section.